subt/ros/proxy/ros_proxy_node.cc

```diff
@@ -47,6 +47,11 @@
 #include <zmq.h>
 #include <assert.h>
 
+extern bool artifactTypeFromString(const std::string& type_text,
```

Comment:
This is just a function declaration. Having it without `extern` in a header file and including that header should be equivalent to having it here without `extern`. Why do you need that keyword?

Reply:
thanks :) ... how many years do you think I was not writing any C/C++ code now? ... and yes, you are right

Reply:
fixed, thanks